sphinx/screens/dashboard/dashboard/src/main/res/values/strings.xml

```diff
@@ -88,6 +88,12 @@
     <string name="dashboard_network_disconnected_node_toast">Disconnected node</string>
     <string name="dashboard_network_connected_node_toast">Connected node</string>
 
+
+    <string name="dashboard_save_profile_popup_title">Save Profile</string>
+    <string name="dashboard_save_profile_button">Submit Profile</string>
+    <string name="dashboard_save_profile_generic_error">Could not Save Profile</string>
```

Comment:
Add strings on Spanish, Japanese and Chinese folders. Add them in english until we get the translations. Otherwise the build process when generating apks will fail.

Reply:
resolved in 187b4f1